.env.example

```
@@ -0,0 +1,20 @@
+# API
+AXIOM_DOMAIN=https://api.axiom.co
+AXIOM_TOKEN=
+BETTER_AUTH_SECRET=
+DATABASE_URL=postgres://postgres:supersecret@localhost:5432/universal-starter?sslmode=require
+DATASET_NAME=universal-
+EMAIL_FROM=noreply@yourapp.com
+EMAIL_FROM_NAME=Your App Name
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)